eth/eventhandler/handlers.go

```diff
@@ -95,6 +95,7 @@ func (eh *EventHandler) handleOperatorAdded(txn basedb.Txn, event *contract.Cont
 	return nil
 }
 
+// This function is currently no-op and it will do nothing. Operator removed event is not used in the current implementation.
```

Comment:
i think we can remove the commented code inside and move this comment there, also clarifying that it will do nothing beyond validating the operator exists